src/components/MovieLibrary.jsx

```diff
@@ -1 +1,36 @@
 // implement MovieLibrary component here
+import React, { Component } from 'react';
+
+import MovieList from './MovieList';
+import SearchBar from './SearchBar';
+import AddMovie from './AddMovie';
+
+class MovieLibrary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTest: '',
+      bookmarkedOnly: false,
+      selectedGenre: '',
+    };
+  }
+
+  render() {
+    const { movies } = this.props;
```

Comment:
Esse _movies_ aqui ele pede que seja declarado no _state_(_initializes `movies` in state with the prop `movies`_), então seria algo tipo:
```suggestion
    movies: props.movies
```